.registryrc

```diff
@@ -1 +1 @@
-01f2271592da4a06962e31ed384201f32cb070ab
+3aeee159382e1bbbf6539627aee52c9059d6bce6
```

Comment:
pull main before merging - because this is an older registry version than we have in main now

Reply:
done